action.yml

```diff
@@ -16,7 +16,7 @@ inputs:
     default: https://cache.nixos.org
   diagnostic-endpoint:
     description: "Diagnostic endpoint url where diagnostics and performance data is sent. To disable set this to an empty string."
-    default: "https://install.determinate.systems/magic-nix-cache/perf"
+    default: "-"
```

Comment:
Why `"-"` over `""`? Does detsys-ts allow the latter as well, at least?

Reply:
Users can already set the value to "" to disable it, so if we make the default "" and change how to disable it ... that'd be crummy.